src/sdl2/video.rs

```diff
@@ -1768,6 +1768,17 @@ impl Window {
             Ok(opacity)
         }
     }
+
+    #[doc(alias = "SDL_FlashWindow")]
+    pub fn flash_window(&mut self, operation: sys::SDL_FlashOperation) -> Result<(), String> {
```

Comment:
probably should just be called `pub fn flash(…)`

Reply:
okay i will change that.